ordered_product/views.py

```diff
@@ -0,0 +1,98 @@
+from .models import OrderedProduct
+from client.models import Client
+from supplier.models import Supplier
+from supplier_product.models import SupplierProduct
+from delivery_location.models import DeliveryLocation
+from product.models import Product
+from django.shortcuts import redirect
+from django.shortcuts import render
+
+
+def order_list(request):
+    if isinstance(request.user, Supplier):
```

Comment:
As we have two different User entities, this will probably repeat itself multiple times (in other apps and PRs as well). 
I'll suggest creating a `decorator` that'll take care of it instead of repeating and maintaining it over and over.
remember - a conditional statement == a test case. 

also, check if Django have a builtin to deal with it (I'm not sure)

Reply:
I deleted all these lines, instead I'm using decorator.